test/erlcloud_ddb_tests.erl

```diff
@@ -19,7 +19,7 @@
 -define(_f(F), fun() -> F end).
 
 -export([validate_body/2]).
-                            
```

Comment:
It looks like my editor cleaned up some trailing white spaces.  Sorry about that.

Reply:
thats fine for tests. not fine for actual code as it pollutes the git history